daras_ai_v2/img_model_settings_widgets.py

```
@@ -465,15 +561,15 @@ def instruct_pix2pix_settings():
-    ]:
-        return
-
+@model_filter(
+    Img2ImgModels.dall_e.name,
+    Img2ImgModels.gpt_image_1.name,
+    Img2ImgModels.instruct_pix2pix.name,
+    Img2ImgModels.nano_banana.name,
+    Img2ImgModels.nano_banana_pro.name,
+    exclude=True,
+)
+def prompt_strength_setting(selected_models: str | list[str] = None):
```

Ruff (0.14.5)

Unused function argument: `selected_models`

(ARG001)